### DISCUSSION TITLE
SSL Doesnt Work

### DISCUSSION THREAD

**Discussion by ChaosAPI-Minecraft** [requarks/wiki#6800] [Help / Questions]
2023-10-29T16:10:01.831Z [MASTER] warn: (LETSENCRYPT) Error: Error: Failed HTTP-01 Pre-Flight / Dry Run.
curl 'http://wiki.ltb.d3vilg4ming.de/.well-known/acme-challenge/test-1b41bc225e2e950204019803f3b3095d-0'
Expected: 'test-1b41bc225e2e950204019803f3b3095d-0.LegTwjRM4oC18e_Sh7AHucPEF43Cv5UEESjJ-wktgDs'
Got: '<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 2.0//EN">
<html><head>
<title>404 Not Found</title>
</head><body>
<h1>Not Found</h1>
<p>The requested URL was not found on this server.</p>
<hr>
<address>Apache/2.4.56 (Debian) Server at wiki.ltb.d3vilg4ming.de Port 80</address>
</body></html>
'
See https://git.rootprojects.org/root/acme.js/issues/4
2023-10-29T16:10:01.831Z [MASTER] error: Error: Failed HTTP-01 Pre-Flight / Dry Run.
curl 'http://wiki.ltb.d3vilg4ming.de/.well-known/acme-challenge/test-1b41bc225e2e950204019803f3b3095d-0'
Expected: 'test-1b41bc225e2e950204019803f3b3095d-0.LegTwjRM4oC18e_Sh7AHucPEF43Cv5UEESjJ-wktgDs'
Got: '<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 2.0//EN">
<html><head>
<title>404 Not Found</title>
</head><body>
<h1>Not Found</h1>
<p>The requested URL was not found on this server.</p>
<hr>
<address>Apache/2.4.56 (Debian) Server at wiki.ltb.d3vilg4ming.de Port 80</address>
</body></html>
'
See https://git.rootprojects.org/root/acme.js/issues/4

**Comment by NGPixel** [comment]
Make sure your Wiki.js installation is accessible from the internet on port 80.
